src/Data/Rational/Unnormalised/Properties.agda

```diff
@@ -374,6 +380,15 @@ p ≤? q = Dec.map′ *≤* drop-*≤* (↥ p ℤ.* ↧ q ℤ.≤? ↥ q ℤ.* 
   ; isGroup = +-0-isGroup
   }
 
++-abelianGroup : AbelianGroup 0ℓ 0ℓ
++-abelianGroup = record
+  { Carrier        = ℚᵘ
+  ; _≈_            = _≃_
+  ; _∙_            = _+_
+  ; ε              = 0ℚᵘ
+  ; _⁻¹            = -_
```

Comment:
You do not need to list these: they will be inferred because of
the value you pass to the `isAbelianGroup` field.

Reply:
Likewise you can also remove them in the `+-0-group` definition